aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -1051,5 +1046,30 @@ TORCH_LIBRARY_IMPL(aten, Meta, m) {
   m.impl("add.Tensor", binary_op_with_scalar_meta);
 }
 
+// Implements C++-like division. Performs true division with float input(s),
+//   and integer division with integer inputs. Note that integer division is
+//   not equivalent to floor division. Integer division truncates its
+//   quotient while floor division, as the name suggests, floors it.
+Tensor _c_style_div(const Tensor& self, const Tensor& other) {
+  Tensor result;
+  auto iter = TensorIterator::binary_op(result, self, other);
+  div_stub(iter.device_type(), iter);
+  return iter.output();
+}
+
+Tensor& _c_style_div_(Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(self, self, other);
+  div_stub(iter.device_type(), iter);
```

Comment:
nit: calling this div_stub doesn't seem great.

Reply:
We could follow-up in a separate pr and change the name.